modules/ztp-nbde-installing-a-tang-server.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-nbde-implementation-guide.adoc
+
+[id="ztp-nbde-installing-a-tang-server_{context}"]
+= Installing a Tang server
+
+.Procedure
+
+. You can install a Tang server on a {op-system-base-full} machine using either of the following commands:
++
+[source,terminal]
+----
+$ yum install tang
+----
++
+[source,terminal]
+----
+$ dnf install tang
```

Comment:
This needs to be run as root so `#` and not `$`, or use sudo:
```
$sudo dnf install tang
```

Reply:
Added sudo